src/components/app.tsx

```diff
@@ -96,6 +97,17 @@ class App extends React.Component<Props, State> {
                     </SignedTokenRefresh>
                   )}
                 />
+                <Route
+                  path="/adminv2"
```

Comment:
this path is a bit misleading. Maybe something like /admin/edit-form/json would be more appropriate